models/balancer/balancer_schema.yml

```diff
@@ -405,4 +405,35 @@ models:
       - name: gauge_address
       - name: pool_id
       - name: project
-      - name: blockchain
\ No newline at end of file
+      - name: blockchain
+
+  - name: balancer_pools_metrics_daily
+    meta:
+      blockchain: arbitrum, avalanche_c, base, ethereum, gnosis, optimism, polygon, zkevm
+      contributors: viniabussafi, metacrypto
+    config:
+      tags: ['arbitrum', 'avalanche_c', 'base', 'ethereum', 'gnosis', 'optimism', 'polygon', 'zkevm', 'balancer', 'pool', 'stats', 'volume', 'tvl', 'fee']
+    description: >
+      This spell aggregates data from the trades, liquidity and protocol fees spells, by day and pool, while also displaying some basic information about the pool
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - block_date
+            - blockchain
+            - project_contract_address
```

Comment:
ideally we match this test with columns in unique_keys of the model. can you update one of them to reflect the true keys?

Reply:
yep, sorry for the oversight